### PR TITLE
Setup pw for component tests and start implement test scenarios

### PR DESCRIPTION
**Pull request by BoronenkoTimurs**
Done:

1. Implement scenarios for admin and user with open space by navigation in the app.
2. Start to setup base for the tests in PW.
3. Added data-testid for links.

### CODE REVIEW

File: .github/workflows/github-pages-deploy.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ name: Test and deploy static content to Pages
 on:
   # Runs on pushes targeting the default branch
   push:
-    branches: ['master']
+    branches: ['master', 'Setup-pw-for-component-unit-testing']
```
Comment (Nikolsonn) [inline .github/workflows/github-pages-deploy.yml:7]: Might be it should removed?

File: package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "lint:fix": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0 --fix",
     "preview": "vite preview",
     "predeploy": "npm run build",
-    "test": "npx playwright test"
+    "test:e2e": "playwright test - c playwright.config.ts",
+    "test:ct": "NODE_ENV=development playwright test -c playwright-ct.config.ts"
```
Comment (Nikolsonn) [inline package.json:16]: What ct means?

File: playwright/tests/e2e/user.spec.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+// TODO: Realize test scanarios for user from MIRO
```
Comment (Nikolsonn) [inline playwright/tests/e2e/user.spec.ts:1]: realize test scenarios?

File: playwright/tests/steps/components/navigation.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import test, { Page } from "@playwright/test";
+
+// TODO: Realize methodes for navigation insead the app
```
Comment (Nikolsonn) [inline playwright/tests/steps/components/navigation.ts:3]: Can you please define what it means?

File: package.json (hunk 2 above)
Comment (BoronenkoTimurs) [reply package.json:16]: "CT" need to run component/unit tests - https://playwright.dev/docs/test-components

File: playwright/tests/e2e/user.spec.ts (hunk 3 above)
Comment (BoronenkoTimurs) [reply playwright/tests/e2e/user.spec.ts]: Markdown for future code. Should I remove two empty files?

Comment (Nikolsonn) [reply playwright/tests/e2e/user.spec.ts]: Remove empty files, or add basic scenario

File: .github/workflows/github-pages-deploy.yml (hunk 1 above)
Comment (BoronenkoTimurs) [reply .github/workflows/github-pages-deploy.yml]: Removed.

**Pull request merged**